test/mri/excludes/TestEncodingConverter.rb

```diff
@@ -7,27 +6,18 @@
 exclude :test_cr_newline2, "needs investigation"
 exclude :test_crlf_newline, "needs investigation"
 exclude :test_crlf_newline2, "needs investigation"
-exclude :test_econv_new_hash, "needs investigation"
+exclude :test_default_external, "needs investigation"
```

Comment:
One new failure? Is this just a case of making it more correct in some areas, but it's still not perfect so it's exposed some more failures in other areas?

Reply:
No. I just removed the quotes in the symbol literal and moved it from the bottom of the file to where it occurs during test execution.